.github/workflows/testsuite.yml

```diff
@@ -2,12 +2,11 @@ name: testsuite
 
 on:
   push:
-    branches:
-      - "*"
-      - '!docs'
     tags-ignore:
-      - "*"
+      - "*"       # Exclude all tags
   pull_request:
+    branches-ignore:
+      - "main"    # Exclude main for PRs as well
```

Comment:
Why are we turning off test runs for the main branch? I want them to run. This seems unrelated to the rest of the commit.

Reply:
It seems counterintuitive it just means the testsuite.yml has rules to run instead of the rules here.    Mai just rearranged the rules.

- testsuite will still run against main per the rules in testsuite.yml
- PRs will still run testsuite as a workflow dispatch from within the openstack yml